gammapy/cube/fit.py

```diff
@@ -172,6 +178,115 @@ def likelihood(self):
 
         return stat
 
+    def to_hdulist(self):
+        """Convert map dataset to list of HDUs.
+
+        Returns
+        -------
+        hdulist : `~astropy.io.fits.HDUList`
+            Map dataset list of HDUs.
+        """
+        # TODO: what todo about the model and background model parameters?
+        exclude_primary = slice(1, 3)
```

Comment:
Maybe put `[1:]` inline to remove the primary HDUs below? Or if you want to keep this variable, I'd suggest to put `slice(1, None)` instead of `slice(1, 3)` to make it clearer that this really is only removing the first element.

Reply:
Done